### ISSUE TITLE
Possible doc issue

### ISSUE THREAD

**Issue by cfjedimaster** [dstromberg2/maze-generator#2]
"So if the first cell in the maze is defined as maze[0][0][0,1,1,0], there would be a wall on the top and the left of that particular cell, with openings on the right and bottom."

Given that it is Top, Right, Left, Bottom wouldn't there be a wall on top and bottom?

**Comment by cfjedimaster** [comment]
Ah, I think you mean top, right, bottom, left, so the readme is wrong, but not the part I said. :)

**Comment by scunliffe** [comment]
Correct, the CSS order is Top, Right, Bottom, Left. The documentation has the last two items accidentally reversed.
